<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filterConvultion</class>
 <widget class="QWidget" name="filterConvultion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>138</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="le22">
     <property name="text">
      <string notr="true">4.0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="le13">
     <property name="text">
      <string notr="true">0.0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="le21">
     <property name="text">
      <string notr="true">-1.0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="le23">
     <property name="text">
      <string notr="true">-1.0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="le11">
     <property name="text">
      <string notr="true">0.0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le12">
     <property name="text">
      <string notr="true">-1.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="le31">
     <property name="text">
      <string notr="true">0.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le32">
     <property name="text">
      <string notr="true">-1.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="le33">
     <property name="text">
      <string notr="true">0.0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
